dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -175,21 +204,27 @@ protected ESIndexAPI(final ESClient esclient, final ContentletIndexAPIImpl iapi,
      * @throws InterruptedException
      * @throws ExecutionException
      */
-    public Map<String, Integer> flushCaches(final List<String> indexNames) throws InterruptedException, ExecutionException {
-      Logger.warn(this.getClass(), "Flushing Elasticsearch index caches:" + indexNames);
-      if(indexNames==null || indexNames.isEmpty()) {
-        return ImmutableMap.of("failedShards",0, "successfulShards", 0);
-      }
-      final Client client = esclient.getClient();
-      final ClearIndicesCacheRequestBuilder requestBuilder =
-          client.admin().indices().prepareClearCache(indexNames.toArray(new String[indexNames.size()]));
-
-      final ClearIndicesCacheResponse res = esclient.getClient().admin().indices().clearCache(requestBuilder.request()).get();
-      Map<String, Integer> map= ImmutableMap.of("failedShards",res.getFailedShards(), "successfulShards", res.getSuccessfulShards());
- 
-      Logger.warn(this.getClass(), "Flushed Elasticsearch index caches:" + map);
-      return map;
-    }
+	public Map<String, Integer> flushCaches(final List<String> indexNames)  {
+		Logger.warn(this.getClass(), "Flushing Elasticsearch index caches:" + indexNames);
+		if(indexNames==null || indexNames.isEmpty()) {
+			return ImmutableMap.of("failedShards",0, "successfulShards", 0);
+		}
+
+		ClearIndicesCacheRequest request = new ClearIndicesCacheRequest(
+				indexNames.toArray(new String[0]));
+
+		ClearIndicesCacheResponse clearCacheResponse = Sneaky.sneak(()->(
+				RestHighLevelClientProvider.getInstance().getClient().indices()
+				.clearCache(request, RequestOptions.DEFAULT)));
+
+
+		Map<String, Integer> map= ImmutableMap.of("failedShards",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'map' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)